### PR TITLE
[CLA] saeed-raeisi signature

### PR DESCRIPTION
**Pull request by saeed-raeisi**
Description of the issue/feature this PR addresses:
https://github.com/odoo/odoo/issues/98025

Current behavior before PR:
[CLA] saeed-raeisi signature

Desired behavior after PR is merged:
[CLA] saeed-raeisi signature

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by saeed-raeisi** [comment]
https://github.com/OCA/l10n-iran/tree/14.0/l10n_ir_states

**Comment by saeed-raeisi** [comment]
@pouyamn

File: addons/l10n_ir_states/data/res.city.csv [hunk 1]
```diff
@@ -0,0 +1,1103 @@
+id,country_id:id,state_id:id,name
+city_ir_13001,base.ir,state_ir_13,تبريز
```
Comment (malb-odoo) [inline addons/l10n_ir_states/data/res.city.csv:2]: should be written in english and then transltated back

File: addons/l10n_ir_states/data/res.country.state.csv [hunk 2]
```diff
@@ -0,0 +1,32 @@
+id,country_id:id,name,code
+state_ir_10,base.ir,استان تهران,10
```
Comment (malb-odoo) [inline addons/l10n_ir_states/data/res.country.state.csv:2]: same for transaltion

**Review by malb-odoo — CHANGES_REQUESTED** [review]
Thanks for your work ! I see that there is a more recent PR on OCA, can you update this one too.
IMO creating a new module for states is a bit overkill, it could be integrated to your previous pr: https://github.com/odoo/odoo/pull/97784. 
You just need to put them in the data folder

Also few comments on translation

**Comment by saeed-raeisi** [comment]
https://github.com/odoo/odoo/pull/97784
close this pr and add data file to l10n_ir

**Pull request closed without merging**